examples/annotate-variants-job.properties

```diff
@@ -0,0 +1,45 @@
+# JOB
+spring.batch.job.names=annotate-variants-job
+
+# SUBMISSION FIELDS
+input.vcf=/path/to/input.vcf
+input.vcf.id=1234
+input.vcf.aggregation=NONE
+
+input.study.name=Study name
+input.study.id=2345
+input.study.type=COLLECTION
+
+input.pedigree=
+input.fasta=/path/to/homo_sapiens/sequence.fa
+
+output.dir=/path/to/output_files
+output.dir.annotation=/path/to/annotation-files/
+output.dir.statistics=/path/to/statistics-files/
```

Comment:
The following output properties shouldn't be necessary:

* output.dir
* output.dir.statistics